src/utils/frontend/file-viewer-helper.js

```diff
@@ -0,0 +1,22 @@
+import { getDocTTLs } from '../network/session-core';
+
+const runCheckFiles = async (session, podUrl) => {
```

Comment:
I'm not sure I understand what this function is for. The server handles checking permissions. In fact, a well-implemented server won't give you enough information to tell if you're unauthorized, or if the document you're trying to fetch doesn't exist.

Just fetch all the documents you can from the server, and display whatever you get back. No need to pass stuff through another check.

Reply:
Think I'll rename this to handleErrorsMessages. The message is used for that primarily based on what is being returned from the getDocTTLs function

Reply:
Actually, I should rename this as `runShowFiles` instead.